### PR TITLE
Correction of spelling errors and typos

### PR DESCRIPTION
**Pull request by princeofnaxos**
Came across some typos and corrected them.

### CODE REVIEW

**Comment by princeofnaxos** [comment]
You're absolutely right, but I couldn't find out how to do that from 
within Sourcetree.

File: lib/Froxlor/Dns/Dns.php [hunk 1]
```diff
@@ -94,6 +94,10 @@ public static function createDomainZone($domain_id, $froxlorhostname = false, $i
 			}
 		}
 
+		// ftp
```
Comment (d00p) [inline lib/Froxlor/Dns/Dns.php]: This does not belong to "correction of spelling errors and typos"....

**Comment by discordier** [comment]
Things are getting worse in here... now we have a merge of origin master and some reverted commits.

@princeofnaxos please do the following:
1. Rename your local master brach to some other name (i.e. old-master)
2. checkout the master branch from origin.
3. create a new branch based on master named `fix-typos` or the like.
4. cherry pick the commits 7ccd8a39760437077b4910dc60f268e233b256f8 and c2fe2e0cbc58ed52732fd4cbdf03861afdc79425
5. push the new branch to github.
6. edit this PR to reflect the new source branch.

**Comment by princeofnaxos** [comment]
Been trying to fix it, but didn't succeed.

**Comment by princeofnaxos** [comment]
I followed discordier's advice. Shall I create a new PR or try and fix this one?

**Comment by d00p** [comment]
just do what @discordier said...it's the correct way. If course you can publish the fixes as new PR as well

**Pull request closed without merging**